packages/ui/src/Navbar/Navbar.tsx

```diff
@@ -0,0 +1,92 @@
+import { ReactNode } from "react";
+import {
+  Box,
+  Flex,
+  Avatar,
+  Link,
+  Button,
+  Menu,
+  MenuButton,
+  MenuList,
+  MenuItem,
+  MenuDivider,
+  useDisclosure,
+  useColorModeValue,
+  Stack,
+  useColorMode,
+  Center,
+} from "@chakra-ui/react";
+import { MoonIcon, SunIcon } from "@chakra-ui/icons";
+import { Logo } from "@crud-beer/ui";
+
+const NavLink = ({ children }: { children: ReactNode }) => (
+  <Link
+    px={2}
+    py={1}
+    rounded={"md"}
+    _hover={{
+      textDecoration: "none",
+      bg: useColorModeValue("gray.200", "gray.700"),
+    }}
+    href={"#"}
+  >
+    {children}
+  </Link>
+);
+
+export default function Navbar() {
+  const { colorMode, toggleColorMode } = useColorMode();
+  const { isOpen, onOpen, onClose } = useDisclosure();
+  return (
+    <>
+      <Box bg={useColorModeValue("gray.900", "gray.900")} px={4} minW="100%">
```

Comment:
same color in light and dark mode?

Reply:
I put it to have the option, because I still didn't have the theme.ts.